packages/canon/src/components/DataTable/mocked-columns.tsx

```diff
@@ -53,22 +54,26 @@ export const columns: ColumnDef<DataProps>[] = [
         href="/"
       />
     ),
+    size: 450,
   },
   {
     accessorKey: 'owner',
     header: 'Owner',
     cell: ({ row }) => (
       <DataTable.TableCellText title={row.getValue('owner')} />
     ),
+    size: 200,
   },
   {
     accessorKey: 'type',
     header: 'Type',
     cell: ({ row }) => <DataTable.TableCellText title={row.getValue('type')} />,
+    size: 150,
   },
   {
     accessorKey: 'tags',
     header: 'Tags',
     cell: ({ row }) => <DataTable.TableCellText title={row.getValue('tags')} />,
+    size: 150,
```

Comment:
Should we use `width` instead?

Reply:
The `size` attribute is coming from Tanstack Table - https://tanstack.com/table/latest/docs/guide/column-sizing